mkosi/__init__.py

```diff
@@ -3259,10 +3259,13 @@ def install_build_src(args: CommandLineArguments, root: str, do_run_build_script
             copy_file(args.build_script, os.path.join(root_home(args, root), os.path.basename(args.build_script)))
 
     sft: Optional[SourceFileTransfer] = None
+    resolve_symlinks: bool = None
```

Comment:
```suggestion
    resolve_symlinks: bool = False
```
The less `None`s the better.

Reply:
Done.